docs/examples/Thumbnail.js

```diff
@@ -0,0 +1,23 @@
+const thumbnailInstance = (
+<Grid>
+	<Row>
+	<Col xs={6} md={3}>
+		<Thumbnail href='#'>
+			<img width={171} height={180} alt='171x180' src='/assets/thumbnail.png' />
+		</Thumbnail>
+	</Col>
+	<Col xs={6} md={3}>
+		<Thumbnail href='#'>
+			<img width={171} height={180} alt='171x180' src='/assets/thumbnail.png' />
+		</Thumbnail>
+	</Col>
+	<Col xs={6} md={3}>
+		<Thumbnail href='#'>
+			<img width={171} height={180} alt='171x180' src='/assets/thumbnail.png' />
+		</Thumbnail>
+	</Col>
+	</Row>
+</Grid>
+);
+
+React.render(thumbnailInstance, mountNode);
```

Comment:
Can you change your tabs to spaces? I know this is a small thing but it keeps with consistency throughout the project. We have an `.editorconfig` file at the repo root. [Editor Config](http://editorconfig.org/) has plugins for the most popular code editors out there, and will ensure that this style is maintained for you.